src/Stoa.ts

```diff
@@ -21,11 +23,31 @@ class Stoa {
             res.send(req.params.address + ':' + req.query.height);
         });
 
+        /**
+         * When a request is received through agora-history push
+         * JSON block data is parsed and stored on each storage.
+         */
         this.stoa.post("/push",
-            (req: express.Request, res: express.Response, next: express.NextFunction) => {
-            console.log(req.body);
-            res.status(200).send();
-        });
+            asyncHandler(async (req: express.Request, res: express.Response, next: express.NextFunction) => {
+            var data: any = req.body;
+            var block: any = JSON.parse(data.block);
+            console.log(block);
+            var blockStorage: BlockStorage = await new BlockStorage(err);
+            await blockStorage.createTable(err);
+            await blockStorage.put(block, err);
```

Comment:
This is creating a new BlockStorage class and then trying to create a table each time a block is received?

Reply:
I think that's the problem.
I need to create a table from the constructor and create a generation status value.

Reply:
@MichaelKim20  is making improvements in this area.
About DB instances,

Reply:
Yes I don't quite understand that myself. Everything should be stored in the same database, and the schema should be initialized when the database is created.